message_app_client/src/components/users_table/ActionButton.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+
+const BASE_URL = "http://localhost:5000";
+const ActionButton = (props) => {
+    const { otherParticipantId, userId } = props;
+    const [ status, setStatus ] = useState(props.status);
+    const [ isSender, setIsSender ] = useState(props.isSender);
+
+    const handleOnclick = async (buttonType) => {
+        if (buttonType === "accept"){
```

Comment:
Since this line, please add a space between `else if`, `if`, and `()`. Please be consistent.